src/edu/wright/cs/raiderplanner/view/UiManager.java

```diff
@@ -553,6 +558,8 @@ public File savePlannerFileDialog() {
 		}
 		fileChooser.setInitialDirectory(savesFolder);
 		File file = fileChooser.showSaveDialog(mainStage);
+		if(file == null && setupCount == 0)System.exit(0);//allows program to close if cancel or exit are pressed

```

Comment:
I am relatively certain that this line will cause CheckStyle to complain. If you fix it, it might be what you need to get the build to pass.